go.mod

```diff
@@ -220,6 +220,8 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // If you change this please also update docs/INSTALL.md and GO_VERSION in
 // Makefile (then run `make lint` to see where else it needs to be updated as
 // well).
-go 1.23.12
+go 1.24.6
```

Comment:
I don't think we'd like to bump the go version here without having bumped it to `1.24` anywhere else right?

Reply:
yeah we bumped the version also for LND separately.